lib/domgen/sql/model.rb

```diff
@@ -385,6 +385,22 @@ def pre_verify
           end
         end
       end
+
+      attr_writer :sql_driver
+
+      def sql_driver
+        if @sql_driver.nil?
+          @sql_driver =
+            if self.repository.facet_enabled?(:pgsql)
+              "org.postgresql.Driver"
+            elsif self.repository.facet_enabled?(:mssql)
+              "net.sourceforge.jtds.jdbc.Driver"
```

Comment:
While it is not yet universal through this code base I try to prefer single quotes as that is the ruby communities latest recomendations